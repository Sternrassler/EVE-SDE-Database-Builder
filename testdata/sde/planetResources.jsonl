{"planetTypeID":2016,"typeID":2267,"quantity":0.15}
{"planetTypeID":2016,"typeID":2268,"quantity":0.20}
{"planetTypeID":2063,"typeID":2270,"quantity":0.35}
