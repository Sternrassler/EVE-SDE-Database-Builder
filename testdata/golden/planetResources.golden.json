[
  {
    "planetTypeID": 2016,
    "resourceTypeID": 0
  },
  {
    "planetTypeID": 2016,
    "resourceTypeID": 0
  },
  {
    "planetTypeID": 2063,
    "resourceTypeID": 0
  }
]